app/csv.rb

```diff
@@ -20,34 +21,55 @@ def each(&blk)
   end
 
   class API
+    @@cache_dir = 'public/cache'
```

Comment:
If we're abstracting it into something that can be defined at the top of the class, should this be a constant instead of a class variable? 
http://rubylearning.com/satishtalim/ruby_constants.html